modules/controlplane/main.tf

```diff
@@ -3,7 +3,49 @@
 # Control Plane
 ######################################################
 
-resource "aws_security_group" "ecs_control_plane_sg" {
+
+locals {
+
+
+
+
+  container_definitions_secrets = concat(
```

Comment:
I think that we revert this change given that it didn't solve the issue

I think it makes it harder to read in this context